{"metadata":{"role":"symbol","extendedModule":"Swift","title":"localizedDescription","platforms":[{"beta":false,"name":"iOS","unavailable":false,"introducedAt":"8.0","deprecated":false},{"introducedAt":"10.10","beta":false,"name":"macOS","deprecated":false,"unavailable":false},{"beta":false,"name":"tvOS","deprecated":false,"unavailable":false,"introducedAt":"9.0"},{"beta":false,"introducedAt":"2.0","name":"watchOS","deprecated":false,"unavailable":false}],"fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"localizedDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"}],"externalID":"s:s5ErrorP10FoundationE20localizedDescriptionSSvp::SYNTHESIZED::s:10SwiftKyber9ExceptionO","modules":[{"name":"SwiftKyber","relatedModules":["Swift"]}],"roleHeading":"Instance Property","symbolKind":"property"},"kind":"symbol","sections":[],"hierarchy":{"paths":[["doc:\/\/SwiftKyber\/documentation\/SwiftKyber","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/Error-Implementations"]]},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"Error.localizedDescription"},{"text":".","type":"text"}],"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"text":" }","kind":"text"}],"platforms":["macOS"]}],"kind":"declarations"}],"schemaVersion":{"patch":0,"minor":3,"major":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/exception\/localizeddescription"]}],"identifier":{"url":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","interfaceLanguage":"swift"},"references":{"doc://SwiftKyber/documentation/SwiftKyber/Exception":{"abstract":[{"type":"text","text":"The Kyber exceptions"}],"title":"Exception","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception","navigatorTitle":[{"kind":"identifier","text":"Exception"}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Exception","kind":"identifier"}],"type":"topic","role":"symbol","kind":"symbol","url":"\/documentation\/swiftkyber\/exception"},"doc://SwiftKyber/documentation/SwiftKyber/Exception/localizedDescription":{"identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/localizedDescription","type":"topic","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}],"abstract":[],"kind":"symbol","url":"\/documentation\/swiftkyber\/exception\/localizeddescription","title":"localizedDescription","role":"symbol"},"doc://SwiftKyber/documentation/SwiftKyber/Exception/Error-Implementations":{"kind":"article","type":"topic","abstract":[],"title":"Error Implementations","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/Error-Implementations","url":"\/documentation\/swiftkyber\/exception\/error-implementations","role":"collectionGroup"},"doc://SwiftKyber/documentation/SwiftKyber":{"kind":"symbol","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber","type":"topic","abstract":[{"text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard","type":"text"}],"title":"SwiftKyber","url":"\/documentation\/swiftkyber","role":"collection"}}}